packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1491,7 +1491,6 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
 
   @override
   void pointerScroll(double delta) {
-    print('delta $delta');
```

Comment:
Thanks

Reply:
I plan to update the pr tonight, so please pause the review.

I'm sorry I accidentally submitted the debug code.